lib/LaTeXML/resources/CSS/LaTeXML.css

```diff
@@ -244,16 +243,19 @@ span.ltx_rowspan { position:absolute; top:0; bottom:0; }
 
 /*======================================================================
   Columns */
-.ltx_page_column1 {
-    width:44%; float:left; } /* IE uses % of wrong container*/
+.ltx_page_columns {
+    display: flex; }
+.ltx_page_column1,
 .ltx_page_column2 {
-    width:44%; float:right; }
-.ltx_page_columns > .ltx_page_column1 {
-    width:48%; float:left; }
-.ltx_page_columns > .ltx_page_column2 {
-    width:48%; float:right; }
-.ltx_page_columns:after {
-    content:"."; display:block; height:0; clear:both; visibility:hidden; }
+    flex:100%; }
+@media screen and (max-width: 80em){
+  .ltx_page_columns {
+     flex-direction: column; }
+  .ltx_page_column1 > ul {
+     margin-bottom: 0; }
+  .ltx_page_column2 > ul {
+     margin-top: 0; }
```

Comment:
Why the special treatment for `ul` ? What about `ol`, other blocky bits? Haven't tested, just looked oddly specific.

Reply:
Yeah, probably ol, dl should be there. Basically the idea is that 2 columns is often used to split a table of contents/index/other list. And these often have top/bottom margins. But when you're putting 2 of those lists back into (the illusion of) one list, you want to remove the margin between them.

Reply:
Love the concept, and thanks for extending. I guess we can always extend more on a need basis, as we read more on our phones.

This is also the type of general issue that I feel very tempted to loop back with the arxiv-vanity folks, who have been doing extensive CSS "reimagining" on top of latexml, but I am not at all certain they have the time/desire to spend time reviewing latexml's CSS. I'll tag @bfirsh just so that he is aware, but no assumptions he has the time.

Reply:
@brucemiller are there strong reasons for not using the native CSS multicolumn here?